sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -273,6 +270,10 @@ def show(*pcolls, **configs):
   The given pcolls can be dictionary of PCollections (as values), or iterable
   of PCollections or plain PCollection values.
 
+  The user can specify either the max number of elements with `n` to read
+  or the maximum duration of elements to read with `duration`. When a limiter is
+  not supplied, it is assumed to be infinite.
+
   There are 2 boolean configurations:
```

Comment:
Nit: how about merging these configurations in the docstrings.
Something like:
```
There are 4 configurations: 
  #. include_window_info=<True/False>. If True, windowing information of the
       data will be visualized too. Default is false.
  #. visualize_data=<True/False>. By default, the visualization contains data
       tables rendering data from given pcolls separately as if they are
       converted into dataframes. If visualize_data is True, there will be a
       more dive-in widget and statistically overview widget of the data.
       Otherwise, those 2 data visualization widgets will not be displayed.
  #. n=<int>. Max number of elements to visualize. Default 'inf'.
  #. duration=<int>. Max duration of elements to read. Default 'inf'.
```

Reply:
Done, added some arg doc strings.